src/subscription_manager/gui/about.py

```diff
@@ -34,6 +34,7 @@
             "Red Hat trademarks are not licensed under GPLv2. No permission is "
             "granted to use or replicate Red Hat trademarks that are incorporated "
             "in this software or its documentation.\n")
+AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
```

Comment:
This only works if you are running subscription-manager-gui from the source checkout.

```
[subscription-manager (vritant/1254460-credits u=)]$ cd
[~]$ sudo subscription-manager-gui 
Traceback (most recent call last):
  File "/sbin/subscription-manager-gui", line 132, in <module>
    from subscription_manager.gui import managergui
  File "/usr/share/rhsm/subscription_manager/gui/managergui.py", line 50, in <module>
    from subscription_manager.gui.about import AboutDialog
  File "/usr/share/rhsm/subscription_manager/gui/about.py", line 37, in <module>
    AUTHORS = [line.strip() for line in open("%s/docs/authors.txt" % os.getcwd(), 'r')]
IOError: [Errno 2] No such file or directory: '/home/adrian/docs/authors.txt'
```